poetry.lock

```diff
@@ -0,0 +1,454 @@
+[[package]]
```

Comment:
Interesting, I guess I've always used it with packages that are not libraries. It does make sense for a library to be more flexible about what it can use because that avoids dependency conflicts.